production/loki-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -82,7 +82,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
     },
 
   jobMatcher(job)::
-    'cluster=~"$cluster", job=~"($namespace)/%s"' % job,
+    'cluster=~"$cluster", job=~"($namespace)|%s"' % job,
```

Comment:
Hmm. I think this may break existing behavior?

with this change, can there still be job="loki-dev/ingester"?

Reply:
@kavirajk the change I added here is because I thought the pattern was to consider either the namespace or the job.
In my case, metrics produced by Loki have the following form:
`loki_ingester_sent_chunks{container="ingester", endpoint="ingester-http-metrics", instance="xxxx:3100", job="ingester", namespace="loki", pod="ingester-0", service="ingester"}`

As you can see, job does not include the namespace at all.
So either the metric Loki emits or the pattern above are broken, I assumed the latter

Reply:
@irizzant the job label isn't coming from loki itself but instead from promtail. we relay on that pattern of `namespace/job`, here's the relevant section of our promtail scrape config:
```
- action: replace
 replacement: $1
 separator: /
 source_labels:
   - __meta_kubernetes_namespace
   - __service__
 target_label: job
```